docs/release-notes/releaseposts/2024042301.md

```diff
@@ -0,0 +1,19 @@
+---
+draft: false
+date: 2024-04-23
+---
+
+# GraFx Publisher Updates
+
+![rn_icon](/assets/icon-GraFx-Publisher.svg)
+
+<!-- more -->
+
+## Improvements
+
+- Strengthened security of REST API
+
+## Fixes
+
+- Fixed issue when drag and drop failed from external assets panel
+- Fixed error related to naming issue during certain API call
```

Comment:
Should be "Fixed error related to naming issue during certain API call**s**" according to Lina's post.

Reply:
Fixed it

Reply:
Test